docs/DeveloperGuide.md

```diff
@@ -400,6 +490,85 @@ Step 7. Finally, it will return a `CommandResult` if the operation is successful
     * Cons: Different OSes have different file paths convention.
 
 
+### Export JSON file
+
+#### Implementation
+
+The export JSON file will export all the current data into a JSON file.
+
+It works by utilizing the same mechanism that is used by AB3 when saving the addressbook into a JSON file.
+
```

Comment:
Potential readers of our DG might not know what AB3 is. Maybe we could replace AB3 with Socius since the saving mechanism is the same?

Reply:
Yeah that's a good idea